src/io/browser-load.js

```diff
@@ -26,17 +26,15 @@ export function loadFile(url, opts) {
  * image.crossOrigin can be set via opts.crossOrigin, default to 'anonymous'
  * returns a promise tracking the load
  */
-export function loadImage(url, opts = {
-  crossOrigin: 'anonymous'
-}) {
+export function loadImage(url, opts) {
   url = pathPrefix ? pathPrefix + url : url;
 
   return new Promise((resolve, reject) => {
     try {
       const image = new Image();
       image.onload = () => resolve(image);
       image.onerror = () => reject(new Error(`Could not load image ${url}.`));
-      image.crossOrigin = opts.crossOrigin;
+      image.crossOrigin = opts && opts.crossOrigin || 'anonymous';
```

Comment:
Nit: I usually add parenthesis when using multiple operators of similar precedence to make it extra clear what order is intended:
```
image.crossOrigin = (opts && opts.crossOrigin) || 'anonymous';
```